doc/sphinx-guides/source/installation/config.rst

```diff
@@ -499,14 +499,14 @@ Logging & Slow Performance
 
 .. _file-storage:
 
-File Storage: Using a Local Filesystem and/or Swift and/or Object Stores and/or Trusted Remote Stores
------------------------------------------------------------------------------------------------------
+File Storage: Using a Local Filesystem and/or Swift and/or Object Stores and/or Trusted Remote Stores and/or Globus Stores
```

Comment:
Ha. At some point we might want a more generic title instead of listing each type. 😄 Plus, Swift probably shouldn't be second forever. It's hardly used.